app/controllers/institutions_controller.rb

```diff
@@ -16,13 +16,14 @@ def index
     institutions = institutions.search_by_sports(@sports) unless @sports.nil?
```

Comment:
How about improving fuzzy search?

Reply:
For one sport the isn't much of a difference but fuzzy search doesn't work well with multiple different sports whereas the default full text search matches the record when any of the sports mentioned match.

Reply:
Sry it was just a first line so i put there general comment. i should have been more clear, sry